ui/src/main/webapp/src/app/shared/js-tree-angular-wrapper.component.ts

```diff
@@ -125,17 +144,21 @@ export class JsTreeAngularWrapperComponent implements OnInit, OnChanges, OnDestr
     }
 
     ngOnDestroy(): void {
+
```

Comment:
Around line 129, I tried the following replacement code:

`
let jsTree = $(this.element).jstree({
            'plugins': plugins,
            'core': {
                data: this.jsTree
            },
            'checkbox': {
                'tie_selection': false,
                'cascade': 'undetermined+down',
                'three_state': false
            }
        });
        jsTree.on('uncheck_all.jstree', () => {
            this.redrawSelection();
        });
`

It isn't obvious to me that this is the "right" solution, but maybe it gets you in the right direction? The problem is that the nodes seem to be reloaded by jstree immediatley after redrawSelection has been called. This seems to result in uncheck_all being called when it shouldn't and then they never get manually reselected.